frontend/public/index.html

```diff
@@ -6,7 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Property Inventory Management System" />
+    <!-- Any updates to this meta tag must be reflected in the applications nginx.conf.template for these changes to take effect in openshift-->
```

Comment:
the index.html here allows us to fail fast during development, so that if someone adds a new call and forgets to update the CSP the call will be blocked. unfortunately I had to include a unsafe-inline here to get the css to show up in webpack, would be nice to get around this. Will circle back if I have time.

Reply:
Would also be lovely if the same csp used in nginx could be used during development, but I don't see an easy way of doing that.